### DISCUSSION TITLE
Why does DTS module config have a wrong type

### DISCUSSION THREAD

**Discussion by Kamahl19** [TanStack/config#81] [General]
I am wondering why on lines https://github.com/TanStack/config/blob/3a40c7e45a368c26d22e724dfc33bdd8e63e2229/src/build/index.js#L31 and https://github.com/TanStack/config/blob/3a40c7e45a368c26d22e724dfc33bdd8e63e2229/src/build/index.js#L51 there are used strings instead of enum values (99 and 1). There is even ts ignore which makes it obviously intentional but no comment or explanation. Thanks!

**Comment by Kamahl19** [comment]
@lachlancollins @tannerlinsley  sorry to bother you by tagging, I am not sure if you are aware of new questions in Discussions. Thanks!

**Reply by lachlancollins** [reply]
Hi @Kamahl19 , the short answer is basically that it worked, and was more readable. Unfortunately with TS 5.5 this is no longer possible, so I've fixed this in #97

**Comment by Kamahl19** [comment]
Thanks!
